packages/govern-contract-utils/test/erc1167-proxy-factory.test.ts

```diff
@@ -0,0 +1,120 @@
+import { expect } from 'chai'
+import { ethers } from 'hardhat'
+import { Signer } from 'ethers'
+import { defaultAbiCoder } from 'ethers/lib/utils'
+import { generateCodeFromContract } from './helpers'
+import {
+  CloneFactoryMockFactory,
+  CloneFactoryMock,
+  ClonedContract,
+  ClonedContractWithInit
+} from '../typechain'
+import
+  * as ClonedContractArtifact
+  from '../artifacts/contracts/test/ClonedContract.sol/ClonedContract.json'
+import
+  * as ClonedContractWithInitArtifact
+  from '../artifacts/contracts/test/ClonedContractWithInit.sol/ClonedContractWithInit.json'
+
+
+describe('ERC1167ProxyFactory', () => {
+  let signers: Signer[],
+    owner: string,
+    factory: CloneFactoryMock,
+    clonedContractAddress: string,
+    clonedContract: any
+
+  before(async () => {
+    signers = await ethers.getSigners()
+    owner = await signers[0].getAddress()
+    const CloneFactoryMock = (await ethers.getContractFactory('CloneFactoryMock')) as CloneFactoryMockFactory
+    factory = await CloneFactoryMock.deploy()
```

Comment:
This seems a common patter across tests. I already did a few helper on the `govern-create` package that we can reuse here. I wonder if maybe the best place is to have them in this repo (`govern-contract-utils`) and export them in other places 🤔